metagpt/provider/openai_api.py

```diff
@@ -327,7 +333,12 @@ def _calc_usage(self, messages: list[dict], rsp: str) -> dict:
                 usage["completion_tokens"] = completion_tokens
                 return usage
             except Exception as e:
-                logger.error("usage calculation failed!", e)
+                prompt_tokens = count_message_tokens(messages,"gpt-3.5-turbo-0613")
```

Comment:
judge the self.model in `TOKEN_COSTS` to simplify to the code.

model = self.model if self.model in TOKEN_COSTS else "gpt-3.5-turbo-0613"
try:
    prompt_tokens = count_message_tokens(messages, self.model)
    xxxx
except xxx
    ...